src/command_modules/azure-cli-find/azure/cli/command_modules/find/custom.py

```diff
@@ -3,114 +3,94 @@
+from azure.cli.core import telemetry as telemetry_core
+from knack.prompting import prompt
+from knack.log import get_logger
+logger = get_logger(__name__)
 
-    try:
-        ix = _get_index(cmd.cli_ctx)
-    except ValueError:
-        # got a pickle error because the index was written by a different python version
-        # recreate the index and proceed
-        _create_index(cmd.cli_ctx)
-        ix = _get_index(cmd.cli_ctx)
-
-    qp = MultifieldParser(
-        ['cmd_name', 'short_summary', 'long_summary', 'examples'],
-        schema=_get_schema()
-    )
-
-    if 'OR' in criteria or 'AND' in criteria:
-        # looks more advanced, let's trust them to make a great query
-        q = qp.parse(" ".join(criteria))
+WAIT_MESSAGE = ['I\'m an AI bot (learn more: aka.ms/aladdinkb); Let me see how I can help you...']
+
+EXTENSION_NAME = 'find'
+
+
+def process_query(cli_term):
+    print(random.choice(WAIT_MESSAGE))
+    response = call_aladdin_service(cli_term)
+
+    if response.status_code != 200:
+        logger.error('[?] Unexpected Error: [HTTP {0}]: Content: {1}'.format(response.status_code, response.content))
     else:
-        # let's help out with some OR's to provide a less restrictive search
-        expanded_query = " OR ".join(criteria) + " OR '{}'".format(criteria)
-        q = qp.parse(expanded_query)
-
-    with ix.searcher() as searcher:
-        from whoosh.highlight import UppercaseFormatter, ContextFragmenter
-        results = searcher.search(q)
-        results.fragmenter = ContextFragmenter(maxchars=300, surround=200)
-        results.formatter = UppercaseFormatter()
-        for hit in results:
-            _print_hit(hit)
+        if (platform.system() == 'Windows' and should_enable_styling()):
+            colorama.init(convert=True)
+
+        answer_list = json.loads(response.content)
+        if (not answer_list or answer_list[0]['source'] == 'bing'):
+            print("\nSorry I am not able to help with [" + cli_term + "]."
+                  "\nTry typing the beginning of a command e.g. " + style_message('az vm') + ".")
+        else:
+            print("\nHere are the most common ways to use [" + cli_term + "]: \n")
+            num_results_to_show = min(3, len(answer_list))
+            for i in range(num_results_to_show):
+                current_title = answer_list[i]['title'].strip()
+                current_snippet = answer_list[i]['snippet'].strip()
+                if current_title.startswith("az "):
+                    current_title, current_snippet = current_snippet, current_title
+                    current_title = current_title.split('\r\n')[0]
+                elif '```azurecli\r\n' in current_snippet:
+                    start_index = current_snippet.index('```azurecli\r\n') + len('```azurecli\r\n')
+                    current_snippet = current_snippet[start_index:]
+                current_snippet = current_snippet.replace('```', '').replace(current_title, '').strip()
```

Comment:
All of the output examples show placeholders using `<this-syntax>`. For consistency with actual in-help examples, these should use `{thisSyntax}` because `<this>` will be rendered invisible in online docs and we don't want people seeing this and writing their examples accordingly.